crypto/cryptlib.c

```diff
@@ -319,14 +319,9 @@ void OPENSSL_die(const char *message, const char *file, int line)
  * pointers to volatile by preserving the reads and writes through them.
  * The latter is not required by the Standard if the memory pointed to
  * is not volatile.
- * Pointers themselves are volatile in the function signature to work
- * around a subtle bug in gcc 4.6+ which causes writes through
- * pointers to volatile to not be emitted in some rare,
- * never needed in real life, pieces of code.
  */
-int CRYPTO_memcmp(const volatile void * volatile in_a,
-                  const volatile void * volatile in_b,
-                  size_t len)
+# undef CRYPTO_memcmp
```

Comment:
The point of this patch is to make the sanitizers see the memcmp() call, and so asan can check the sizes, and the fuzzers can also get to see the data.